pythonfmu/builder.py

```diff
@@ -317,6 +335,8 @@ def build_FMU(
                 zip_fmu.writestr(
                     "modelDescription.xml", xml_str.toprettyxml(encoding="UTF-8")
                 )
+            if newargs is not None:
+                sys.modules.pop(Path(script_file).stem)  # otherwise old script may be active when loading the FMU!
```

Comment:
module syntax can be `aaa.bbb.ccc.` Path.stem seems to only return filename without the final suffix.